indra/newview/llfloatersidepanelcontainer.h

```diff
@@ -89,6 +96,8 @@ class LLFloaterSidePanelContainer : public LLFloater
 
 protected:
     void onCloseMsgCallback(const LLSD& notification, const LLSD& response);
+
+	LLPanel* mMainPanel = nullptr;
```

Comment:
```suggestion
    LLPanel* mMainPanel = nullptr;
```
for [pre-commit](https://github.com/secondlife/git-hooks)